Source/WebCore/css/CSSFontSelector.cpp

```diff
@@ -351,6 +380,7 @@ FontRanges CSSFontSelector::fontRangesForFamily(const FontDescription& fontDescr
     };
 
     const auto& fontPaletteValues = lookupFontPaletteValues(familyName, fontDescription);
+    Ref<FontFeatureValues> fontFeatureValues = lookupFontFeatureValues(familyName);
```

Comment:
It’s really unfortunate that we have designed an interface that always allocates these on the heap, and requires allocating a destroying an empty one just to express "there are no values". We could use `RefPtr` instead and use nullptr for when there are none, or maybe not heap-allocate `FontFeatureValues`?

Reply:
`FontFeatureValues` is big and thus kinda need to be heap-allocated (for example, there is a static_assert related to key of HashMap being small than its size).

Indeed, we need to express the fairly frequent "there is no value" : I've changed the patch to use `RefPtr` instead of `Ref` in most places.